package.json

```diff
@@ -57,10 +57,12 @@
     "invariant": "^2.0.0"
   },
   "npmName": "redux",
-  "npmFileMap": [{
-    "basePath": "/dist/",
-    "files": [
-      "*.js"
-    ]
-  }]
+  "npmFileMap": [
+    {
```

Comment:
npm itself autoformatted this. Makes sense to keep it so the next time you don't need to revert changes.